### DISCUSSION TITLE
Requesting Access to Contribute

### DISCUSSION THREAD

**Discussion by AdamRosas27** [wilfredinni/python-cheatsheet#302] [General]
Hey there,
I've been trying to push a change to "origin fix_bug" for the python-cheatsheet  but ran into a "Permission denied" issue. Is there a way I can request access to contribute? I'm eager to get involved and think I can bring value to the project. Thanks!

**Comment by Jdubbbbb** [comment]
Hi Adam,

Check out these links.

https://softwareengineering.stackexchange.com/questions/67192/should-i-release-source-code-to-fix-bug

https://stackoverflow.com/questions/40883743/how-to-fix-a-bug-in-a-branch-after-it-has-already-been-merged-with-master

https://stackoverflow.com/questions/51550890/when-i-do-a-git-pull-origin-it-gives-a-permission-denied-how-can-i-resolve-t

Jessica


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Tue, Sep 12, 2023 at 1:20 AM Adam Rosas ***@***.***> wrote:

> Hey there,
> I've been trying to push a change to "origin fix_bug" for the
> python-cheatsheet but ran into a "Permission denied" issue. Is there a way
> I can request access to contribute? I'm eager to get involved and think I
> can bring value to the project. Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wilfredinni/python-cheatsheet/discussions/302>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ3EA6TGS7PARO7ZCOOWK33XZ7WJPANCNFSM6AAAAAA4UJL2U4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
________
Jessica Weber

 "In order to change the world, you have to get your head together first.”
-Jimi Hendrix

“A small group of thoughtful people could change the world. Indeed, it's
the only thing that ever has.” -Margaret Mead
Sender verifies this email.
